.github/workflows/lol-hackathoon.yml

```
@@ -0,0 +1,310 @@
+name: MASSIVE Restart temp-fe Deployment on Staging
+
+on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)